### ISSUE TITLE
Range cannot be changed via the CHARACTER_STATS feature

### ISSUE THREAD

**Issue by StrangeOne101** [IsaacScript/isaacscript#75]
# Description
In my testing, it seems that range is unable to be modified using the character stats feature.

# Code to Reproduce

```ts
const features = [ISCFeature.CHARACTER_STATS] as const;
const mod = upgradeMod(modVanilla, features);
const parikuDefaultStats = new Map<CacheFlag, number>([
    [CacheFlag.DAMAGE, 3],
    [CacheFlag.SPEED, 0.9],
    [CacheFlag.RANGE, 5 * 40]
 ]);
 mod.registerCharacterStats(PARIKU_TYPE, parikuDefaultStats);
 ```

In game, it can be noted that speed and damage are updated properly. But range is not.
![image](https://github.com/IsaacScript/isaacscript/assets/2769013/d664e6ee-a299-406f-9197-f4fb81b02942)

I have tried with values of `5` as well to see if it was automatically multiplied by 40 for tile width, but that also leaves no change.

# Environment 
- IsaacScript v4.3.0
- isaacscript-common: 87.2.2
- isaacscript-common-node: 4.0.3
- isaacscript-common-ts: 16.1.0
- Isaac Build: Not sure how to find the version, but latest as of 13/03/2024

**Comment by Zamiell** [comment]
ill check it out

**Comment by Zamiell** [comment]
fixed in 35dbceb03cb135014341495ff9411ed7377b91f7

**Comment by StrangeOne101** [comment]
You're amazing. Thanks for such a fast response 😄

**Comment by Zamiell** [comment]
released version 87.3.0, you can try on that one
